conversions/hex-bin.py

```diff
@@ -0,0 +1,44 @@
+#convert hexadecimal into binary
+from doctest import testmod
+import math
+
+
+hex_str = input("").strip()
```

Comment:
Also, you don't need to take inputs, as we are trying to implement the function only, which is guaranteed to be passed with certain arguments.